AssemblerLib/LocalDataInitializer.h

```diff
@@ -146,19 +160,19 @@ class LocalDataInitializer
 #if (OGS_ENABLED_ELEMENTS & ENABLED_ELEMENT_TYPE_LINE) != 0 \
     && OGS_MAX_ELEMENT_DIM >= 0 && OGS_MAX_ELEMENT_ORDER >= 1
         _builder[std::type_index(typeid(MeshLib::Point))] =
-            [](){ return new LAData<NumLib::ShapePoint1>; };
+                OGS_NEW_LOCAL_ASSEMBLER(NumLib::ShapePoint1);
```

Comment:
Actually the lambda was used here, because it is so short. Now the function is a little longer and is moved out. But then there is no reason to use a macro; a template function will do as well.